.env.dist

```diff
@@ -0,0 +1,23 @@
+# This file is a "template" of which env vars needs to be defined in your configuration or in an .env file
+# Set variables here that may be different on each deployment target of the app, e.g. development, staging, production.
+# https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
+
+###> symfony/framework-bundle ###
+APP_ENV=dev
+APP_DEBUG=1
+APP_SECRET=EDITME
+###< symfony/framework-bundle ###
+
+###> doctrine/doctrine-bundle ###
+# Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
+# For a sqlite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
+# Set "serverVersion" to your server version to avoid edge-case exceptions and extra database calls
+DATABASE_URL=mysql://root@127.0.0.1/sylius_${APP_ENV}?serverVersion=5.5
```

Comment:
`serverVersion` should not be set here. See https://github.com/symfony/recipes/blob/cb6ea10b7bb60c4bd9c39ca3e92fb53e7ccce7e5/doctrine/doctrine-bundle/1.6/config/packages/doctrine.yaml#L12

Reply:
https://github.com/symfony/recipes/pull/258